src/sagemaker/jumpstart/types.py

```diff
@@ -2482,3 +2482,17 @@ def __init__(
         self.source_uri = source_uri
         self.model_card = model_card
         self.accept_eula = accept_eula
+
+
+class JumpStartInternalMetadata(JumpStartDataHolderType):
```

Comment:
This is the data class for a Model, correct?

I would rename `JumpStartModelInternalConfig` or `JumpStartModelInternalSettings`